src/app/components/AlertMessage/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import i18next from 'i18next';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  message: string;
+  type: 'error' | 'success' | 'warning';
+}
+
+function AlertMessage({ message, type }: Props) {
+  const getTitle = () => {
+    switch (type) {
+      case 'error':
+        return i18next.t('AlertMessage:titleError');
+      case 'success':
+        return i18next.t('AlertMessage:titleSuccess');
+      case 'warning':
+        return i18next.t('AlertMessage:titleWarning');
+      default:
+        return '';
+    }
+  };
```

Comment:
move this to a `utils.ts` in this folder whicih you wiill send `type` as param and return the message in order to avoid creating a memory reference for this function on each render